helpers/settings.go

```diff
@@ -117,10 +120,10 @@ func (s *Settings) InitSettings(envVars *EnvVars, env *cfenv.App) (retErr error)
 	s.LoginURL = envVars.MustString(LoginURLEnvVar)
 	s.UaaURL = envVars.MustString(UAAURLEnvVar)
 	s.LogURL = envVars.MustString(LogURLEnvVar)
-	s.PProfEnabled = envVars.Bool(PProfEnabledEnvVar)
+	s.PProfEnabled = envVars.MustBool(PProfEnabledEnvVar)
```

Comment:
this doesn't need to be a `mustbool`

Reply:
The way that we've built `MustBool` is that you don't actually have to have the env var set. If you don't set it, it will just default to false. This is similar to how the `flags` package works. However, if you _do_ set it, it must be a valid boolean otherwise you'll get a panic. So, really, the "must bool" is saying "if you set a value for this environment variable it MUST be a valid parsable boolean".

In so many words, here's the doc comment from `MustBool`:

> MustBool gets the boolean environment variable with the given name, if set.
> If not set, false is returned. This matches behavior one would often see with
> command line boolean arguments: if you don't set it, it defaults to false.
> It will panic with an error if it is not a valid boolean.

cc @aeijdenberg